.packit.yaml

```diff
@@ -0,0 +1,12 @@
+specfile_path: tests.spec
+
+upstream_package_name: tests
+downstream_package_name: tests
+
```

Comment:
Maybe an easier option using [actions](https://packit.dev/docs/actions/) to overwrite the default workflow:


```suggestion

actions:
  fix-spec-file: "true"

```